### PR TITLE
Integrate Figma with the storybook

### LINKED ISSUE #153: Integrate Figma with the storybook
**Description**
The Figma file for design library contains a lot of instructions on the usage and development of more components. The contents of the Figma file make a valuable addition to the Storybook. A simple plugin embeds portions of the file in the storybook, so integrating it should meet our needs.

**Additional context**
- [Storybook addon](https://github.com/pocka/storybook-addon-designs)

### PR DESCRIPTION
**Pull request by Tanuj22**
**Fixes**

Fixes #153  by @dhruvkb 

**Description**
[Storybook addon](https://github.com/pocka/storybook-addon-designs) added.

**Type of PR** 
This PR is a feature.

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ module.exports = {
   addons: [
     '@storybook/addon-knobs',
     '@storybook/addon-backgrounds',
-    '@storybook/addon-docs'
+    '@storybook/addon-docs',
+    'storybook-addon-designs/register'
```
Comment (dhruvkb) [inline .storybook/main.js]: The `/register` part is not required.
```suggestion
    'storybook-addon-designs'
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
I'm sorry if the issue description wasn't clear, integrating the addon is not enough. Please also add the [Figma pages](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=1433%3A0) to the corresponding stories.

**Comment by Tanuj22** [comment]
> I'm sorry if the issue description wasn't clear, integrating the addon is not enough. Please also add the [Figma pages](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=1433%3A0) to the corresponding stories.

Sorry for that. On it.

**Comment by Tanuj22** [comment]
There are some `button` which are available in big,middle and small in Figma which but vocabulary has only one. Which do I use in vocabulary?

**Comment by Tanuj22** [comment]
@dhruvkb  Please check the figma pages added. Please suggest changes for these queries -

- Some `buttons` and `Logos` were not available in Figma. 

- Some components were not seperable like in `Typography` and `GithubCorner` so currently I have added them in a single story. 

- What should I do with `Colors`? Should I put all the `Colors` in a single story?

**Review by dhruvkb — APPROVED** [review]
Looks good to me, might conflict with #161.

**Comment by dhruvkb** [comment]
@Tanuj22 could you please pull `master` into this branch? That's the only small thing blocking the PR.

**Comment by Tanuj22** [comment]
@dhruvkb Done

**Comment by dhruvkb** [comment]
@Tanuj22 by the time I came to review, it's out of date again.

**Comment by Tanuj22** [comment]
@dhruvkb  DOne

**Comment by dhruvkb** [comment]
Waiting for PR #161 to be merged. Then this will be of great help.

**Pull request merged**